components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/FeatureDAOImpl.java

```diff
@@ -261,6 +261,28 @@ public boolean deleteFeaturesOfProfile(Profile profile) throws FeatureManagerDAO
         }
     }
 
+    @Override
+    public boolean deleteFeaturesOfProfile(int profileId) throws FeatureManagerDAOException {
```

Comment:
This boolean is used just to acknowledge whether the operation is successful. It returns true if there are any affected rows of the query.